src/test/java/ch/vorburger/exec/ManagedProcessTest.java

```diff
@@ -133,6 +133,15 @@ public void testWaitForWrongMessageIfAlreadyTerminated() throws Exception {
         p.startAndWaitForConsoleMessageMaxMs("some console message which will never appear", 1000);
     }
 
+    @Test
+    public void testProcessExitsGracefullyWhenFinishesBelowMaxWaitTime() throws ManagedProcessException {
```

Comment:
How about instead of introducing an entire new test, instead just add assertThat(process.wasDestroyed(), is(false)); to the existing testSelfTerminatingExec() ?

Reply:
I see you've added it to `testSelfTerminatingExec()` but kept this `testProcessExitsGracefullyWhenFinishesBelowMaxWaitTime()` .. was that intentional? I feel this new test doesn't really add any additional functional coverage, or am I missing something? 

Another problem is that this will fail for anyone build this library on Windows (which believe it not someone actually complained about, back in the day when this code was part of MariaDB4j, someone contributed a PR to make it pass on Windows). The existing test handle that (see `someSelfTerminatingExec()`), so unless you think this really adds value and want to do something e.g. like https://serverfault.com/a/432323/314154, perhaps it's easiest to just remove this test after all?